src/main/java/org/sonar/plugins/checkstyle/CheckstyleSensor.java

```diff
@@ -34,7 +33,7 @@ public CheckstyleSensor(CheckstyleExecutor executor) {
 
     @Override
     public void describe(SensorDescriptor descriptor) {
-        descriptor.onlyOnLanguage(Java.KEY).name("CheckstyleSensor");
+        descriptor.onlyOnLanguage("java").name("CheckstyleSensor");
```

Comment:
This comes from `sonar-java-plugin` and so we have to hardcode it since we want to remove that dependency.

Reply:
````
public class Java extends AbstractLanguage {

  /**
   * Java key
   */
  public static final String KEY = "java";
````